.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,24 +16,9 @@ The status of your PR. ie WIP or Finished.
 Ex: Finished; Pending Review 
 -->
 
-### Changelog
-
-<!-- 
-Details of what was changed goes here.
-Format is :
-- Change 1
-- Change 2
--->
-
-### Checklist <!-- Delete this if you don't have a checklist -->
-<!-- 
-To make a checklist:
-- [] Entry 1
-- [] Entry 2
--->
-
-### Related Issues <!-- Delete this if you don't have any issues related to your PR -->
+### Additional Notes <!-- Omit this if you don't have any issues related to your PR -->
 
-<!-- If you have any issues related to this PR, link them here.
+<!--
+Any Additional notes regarding your PR goes here.

```

Comment:
this section is self explanatory, replace it by saying that they can close issues by using something like `resolves #589`